lib/Backends/Interpreter/InterpreterFunction.h

```diff
@@ -54,6 +54,17 @@ class InterpreterFunction final : public CompiledFunction {
   /// \name CompiledFunction interface
   ///@{
   ~InterpreterFunction() override;
+  /// Does any needed initialization work for the Backend. Adds references to
+  /// constants in externalTensors_.
+  void setupRuns();
```

Comment:
`override` for all these methods.

Reply:
I'd need to add them to the base class first right? That's planned for the next task. I'll align all 3 backends to this api and add it to the base class.

Reply:
The reason I assumed that those are already defined in the CompiledFunction, because
these methods are in the section of 
```
/// \name CompiledFunction interface
  ///@{
```

It's fine as we are going to fix this very soon.